html/admin/api.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
For lack of a better place in the review, I can't get any of the config tasks (Hostname, SSID, Channel, TX Power) to actually change any config. I get a failure (a cross) for all but shutdown and reboot (and they don't actually shutdown or reboot the machine - I can see that they don't wait for the command to complete, hence the "success"). In `/var/log/auth.log` I see:

```
Dec  3 20:44:23 raspberrypi sudo: pam_unix(sudo:auth): conversation failed
Dec  3 20:44:23 raspberrypi sudo: pam_unix(sudo:auth): auth could not identify password for [www-data]
Dec  3 20:44:24 raspberrypi sudo: www-data : user NOT in sudoers ; TTY=unknown ; PWD=/var/www/biblebox/biblebox_default/content/admin ; USER=root ; COMMAND=/usr/local/biblebox/bin/BBoxManage.sh unmountusb
```